config/default_records/request_types/014_limber_scrna_core_cdna_prep_request_types.wip.yml

```diff
@@ -47,13 +41,3 @@ limber_scrna_core_chromium_gem_x_5p_aggregation:
   acceptable_purposes:
     - LRC GEM-X 5p cDNA PCR XP
```

Comment:
As mentioned in Limber PR, inconsistent naming between library type and plate purposes/request types.
5p or 5 prime GEM-X vs GEM-X 5p